ViewLayer/PowerUI/I_PowerUI.h

```diff
@@ -32,53 +32,53 @@ class I_PowerUI
     virtual QLabel& validPacketsInLastMinute() = 0;
     virtual QLabel& invalidPacketsInLastMinute() = 0;
 
-    virtual QLabel& setSetSpeed() = 0;
-    virtual QLabel& setSetCurrent() = 0;
-    virtual QLabel& setActualSpeed() = 0;    
-    virtual QLabel& setBusCurrent() = 0;
-    virtual QLabel& setBusVoltage() = 0;
-    virtual QLabel& setBatteryCurrent() = 0;
-    virtual QLabel& setBatteryVoltage() = 0;
+    virtual QLabel& setSpeed() = 0;
+    virtual QLabel& setCurrent() = 0;
+    virtual QLabel& actualSpeed() = 0;
+    virtual QLabel& busCurrent() = 0;
+    virtual QLabel& busVoltage() = 0;
+    virtual QLabel& batteryCurrent() = 0;
+    virtual QLabel& batteryVoltage() = 0;
 
-    virtual QLabel& setBatteryCMU1Temp() = 0;
-    virtual QLabel& setBatteryCMU1Cell1Voltage() = 0;
-    virtual QLabel& setBatteryCMU1Cell2Voltage() = 0;
-    virtual QLabel& setBatteryCMU1Cell3Voltage() = 0;
-    virtual QLabel& setBatteryCMU1Cell4Voltage() = 0;
-    virtual QLabel& setBatteryCMU1Cell5Voltage() = 0;
-    virtual QLabel& setBatteryCMU1Cell6Voltage() = 0;
-    virtual QLabel& setBatteryCMU1Cell7Voltage() = 0;
-    virtual QLabel& setBatteryCMU1Cell8Voltage() = 0;
+    virtual QLabel& batteryCMU1Temp() = 0;
```

Comment:
In a lot of style guides with camel case generally only the first letter of the acronym is capitalized. So it would be batteryCmu1Temp. This is my favourite style guide: http://geosoft.no/development/cppstyle.html#Naming%20Conventions refer to rule 9 of that section. It is up to you to decide going forward what we should do but I thought I would bring it up for discussion.